src/main/java/seedu/address/model/person/Person.java

```diff
@@ -21,19 +21,21 @@
     private final Email email;
 
     // Data fields
-    private final Address address;
+    private final Address address = null;
     private final Set<Tag> tags = new HashSet<>();
+    private boolean isDone;
 
     /**
      * Every field must be present and not null.
      */
-    public Person(Name name, Phone phone, Email email, Address address, Set<Tag> tags) {
-        requireAllNonNull(name, phone, email, address, tags);
+    public Person(Name name, Phone phone, Email email, boolean isDone) {
+        requireAllNonNull(name, phone, email, isDone);
         this.name = name;
         this.phone = phone;
         this.email = email;
-        this.address = address;
-        this.tags.addAll(tags);
+        this.isDone = isDone;
+        //this.address = address;
+        //this.tags.addAll(tags);
```

Comment:
Hmm. Don't u think that these comments just makes the code look messy?

Reply:
just remove bah. if we want to find out wat used to be there, we can refer to past versions

Reply:
ok